packages/migrations/src/actions/2024.06.11T10-10-00.ms-teams-webhook.ts

```diff
@@ -0,0 +1,8 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2024.06.11T10-10-00.ms-teams-webhook.ts',
+  run: ({ sql }) => sql`
+    ALTER TYPE alert_channel_type ADD VALUE 'MSTEAMS_WEBHOOK';
```

Comment:
Is altering a type in Postgres a safe operation? Will it have some side-effects like rewriting rows in the table or similar?

Reply:
yes absolutely. It does not rewrite rows or anything. This should run in under 100 ms on any DB size.